extensions/scheduler/deployment/src/main/resources/dev-templates/schedules.html

```diff
@@ -1,6 +1,22 @@
 {#include main}
   {#title}Schedules{/title}
   {#body}
+  {#if info:scheduler.running}
+  <form method="post" enctype="application/x-www-form-urlencoded">
+    Status: 
+    <strong>RUNNING</strong> 
+    <input type="hidden" name="action" value="pause">
+    <input id="pause" type="submit" value="Pause" class="btn btn-primary btn-sm">
```

Comment:
Maybe we could add Play / Pause icons to the buttons? I think I did add icons elsewhere, maybe in the deployment pages?